doc/user/content/sql/create-type.md

```diff
@@ -114,6 +118,18 @@ SELECT '{a=>{a=>1}}'::int4_map_map::text AS custom_nested_map;
 {a=>{a=>1}}
 ```
 
+### Custom `row` type
+```sql
+CREATE TYPE row_type AS (a int, b text);
+SELECT ROW(1, 'a')::row_type;
+```
+
+### Nested `row` type
+```sql
+CREATE TYPE nested_row_type AS (a row_type, b float8);
+SELECT ROW(ROW(1, 'a'), 2.3)::nested_row_type;
+```
```

Comment:
Suggestions go haywire with the ```, so leaving the suggested edits as a comment. To make the examples consistent with the rest of the page:

\### Custom `row` type

\```sql
CREATE TYPE row_type AS (a int, b text);

SELECT ROW(1, 'a')::row_type AS custom_row_type;
 \```
 \```
 custom_row_type
\-----------------
 (1,a)
\```

\### Nested `row` type
\```sql
CREATE TYPE nested_row_type AS (a row_type, b float8);

SELECT ROW(ROW(1, 'a'), 2.3)::nested_row_type AS custom_nested_row_type;
\```
\```
 custom_nested_row_type
\------------------------
 ("(1,a)",2.3)
\```